CMakeLists.txt

```diff
@@ -328,6 +328,35 @@ if(NOT VELOX_BUILD_TEST_UTILS
   add_definitions(-DVELOX_DISABLE_GOOGLETEST)
 endif()
 
+# On macOS, search Homebrew for keg-only versions of Bison and Flex. Xcode does
```

Comment:
can we also support the bison/flex provided by Conda? Since this would avoid global package installment. cc @kgpai